tests/ui_tests/app/page/bundle_list/page.py

```diff
@@ -74,3 +74,9 @@ def accept_licence(self, row_num: int = 0):
         self.wait_element_visible(BundleListLocators.LicensePopup.block)
         self.find_and_click(BundleListLocators.LicensePopup.agree_btn)
         self.wait_element_hide(BundleListLocators.LicensePopup.block)
+
+    def click_bundle_in_row(self, row_num: int = 0):
+        """Click on bundle name"""
+        row = self.table.get_row(row_num)
```

Comment:
I guess it's better to divide finding the row and clicking on its child (for example in case you'll need to find row by bundle name)

Reply:
We still can get a row, find the child, and click on its field.
Here it's just for convenience and readability.
Or am I got your idea wrong?

Reply:
Okay, it's for row search reasons. Got it